cli/internal/run/dry_run.go

```diff
@@ -249,6 +250,13 @@ func displayDryTextRun(ui cli.Ui, tasksRun []hashedTask, packagesInScope []strin
 		fmt.Fprintln(w, util.Sprintf("  ${GREY}Log File\t=\t%s\t${RESET}", task.LogFile))
 		fmt.Fprintln(w, util.Sprintf("  ${GREY}Dependencies\t=\t%s\t${RESET}", strings.Join(dependencies, ", ")))
 		fmt.Fprintln(w, util.Sprintf("  ${GREY}Dependendents\t=\t%s\t${RESET}", strings.Join(dependents, ", ")))
+
+		if resolvedTask, err := json.MarshalIndent(task.ResolvedTaskDefinition, "", "\t"); err != nil {
+			// If unmarshalling fails, don't fail the whole thing
+		} else {
+			fmt.Fprintln(w, util.Sprintf("  ${GREY}Resolved Task Definition\t=\t%s\t${RESET}", resolvedTask))
```

Comment:
The logging here is not great. the closing `}` doesn't indent all the way. Need to log this in a table format to make it look better